spago.dhall

```diff
@@ -15,6 +15,6 @@
   ]
 , packages = ./packages.dhall
 , sources = [ "src/**/*.purs" ]
-, license = "BSD-3-Clause"
+, license = "BSD-2-Clause"
```

Comment:
The license identifier used here and in `bower.json` indicated that the license was the BSD 3-Clause license, whereas the license file is only the BSD 2-Clause.